packages/vite/src/node/optimizer/esbuildDepPlugin.ts

```diff
@@ -135,6 +136,24 @@ export function esbuildDepPlugin(
         }
       )
 
+      const rawUrlRE = /\?raw$/
+
+      build.onResolve(
+        { filter: rawUrlRE },
+        async ({ path: id, importer, kind }) => ({
+          path: await resolve(id.replace(rawUrlRE, ''), importer, kind),
+          namespace: 'raw'
+        })
+      )
+
+      build.onLoad({ filter: /.*/, namespace: 'raw' }, async ({ path }) => {
```

Comment:
**suggestion:** It may be that you want `/.+/` instead of `/.*/` cause `.*` means also matching `//`

Reply:
Other lines use the same regex 👇 
https://github.com/aleclarson/vite/blob/28e427a5267296e0aa777d8db4db96792c3bf7a3/packages/vite/src/node/optimizer/esbuildDepPlugin.ts#L166

Reply:
IMO this doesn't make the other lines better :sweat_smile: 
Maybe we should change these also?